tests/static-analysis/reduce.php

```diff
@@ -6,15 +6,19 @@
 
 use loophp\collection\Collection;
 
-function reduce_takeInt(int $int): void
+function reduce_takeInt(?int $int): void
```

Comment:
hmm seems like this one is now the same as `reduce_takeIntOrNull` so perhaps not needed?

Reply:
Indeed! I will update the pr asap